styleguide/source/assets/css/scss/generic/_variables.scss

```diff
@@ -44,6 +44,9 @@ $blue-lighter: #d9f2fc;
 
 $green: #64a70b;
 
+// Custom color variables
+
+$rule:			$gray-8;
```

Comment:
So, separate from your point about horizontal-rules, I added the $rule variable because of color variable usage I noticed in Corp when stripping out all the Chroma stuff. This is probably part of a bigger rethinking of how we organize colors overall, and I really haven't figured that all out yet.